Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -380,9 +380,11 @@ def audit_license
 
         return unless @online
 
-        user, repo = get_repo_data(%r{https?://github\.com/([^/]+)/([^/]+)/?.*}) if @new_formula
+        user, repo = get_repo_data(%r{https?://github\.com/([^/]+)/([^/]+)/?.*})
         return if user.blank?
 
+        return if formula.versioned_formula?
```

Comment:
Maybe move this up to be with the `return unless @online`? No need to fetch repo data if we already know we won't use it.